<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTest1Class</class>
 <widget class="QMainWindow" name="QtTest1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTest1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>96</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hellp World!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>2400</width>
      <height>1000</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>11</y>
      <width>522</width>
      <height>194</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="inPut1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="inPut2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="spinResult">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuutf_8">
    <property name="title">
     <string>Code</string>
    </property>
    <addaction name="actionopen"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionNeaL"/>
   </widget>
   <addaction name="menuutf_8"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNeaL">
   <property name="text">
    <string>NeaL</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtTest1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>inPut1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtTest1Class</receiver>
   <slot>inPut1_ValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inPut2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtTest1Class</receiver>
   <slot>inPut2_ValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inPut1_ValueChanged(int)</slot>
  <slot>inPut2_ValueChanged(int)</slot>
 </slots>
</ui>
